far/FarEng.hlf.m4

```diff
@@ -3364,47 +3364,56 @@ the contents of the file on the disk.
 the external program will be lost.
 
 @CodePagesMenu
-$ #Code pages menu#
-    This menu allows code page selection in the editor and viewer.
+$ #Code page selection menu#
+    This menu allows to select code page in the editor and viewer.
 
-    The menu is divided into several parts:
+    The menu is divided into several sections:
 
-    #Automatic detection# - Far tries to autodetect the code page of the text;
+        #Automatic# - Far will try to autodetect
```

Comment:
Maybe `^<wrap>` here too? Or it won't work because of multiple lines on the left? (same for FarRus.hlf.m4)

Reply:
Yeah! It wraps the first line and then puts the second line of the left column. I am thinking about adding new directive ^<col> (or something like this) but it's a different story.